### PR TITLE
add the materialize library to use it and create the main menu

### PR DESCRIPTION
**Pull request by archisss**
ticket #21 on waffle

captura de pantalla 2018-11-22 a la s 07 25 56

### CODE REVIEW

File: app/views/welcome/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+  <footer class="page-footer grey darken-4">
```
Comment (8geonirt) [inline app/views/welcome/_footer.html.erb]: Always check for the indentation, it should be consistent 🤓, please indent this file correctly.

File: app/views/welcome/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+  <nav>
```
Comment (8geonirt) [inline app/views/welcome/_header.html.erb]: Same for this, once this changes are done I'll approve it, nice job

**Comment by fernandoamz** [comment]
LGTM!!!

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -15,7 +15,19 @@
 
     <p class="notice"><%= notice %></p>
     <p class="alert"><%= alert %></p>
-
+materialize
```
Comment (8geonirt) [inline app/views/layouts/application.html.erb]: Please fix this code, I think you had conflicts and you didn't solve it when merging or you solve them but in the wrong way, I'm pretty sure this `materialize` text doesn't need to be here, also the `navbar-text` class belongs to the header isn't it?

File: app/assets/javascripts/application.js [hunk 4]
```diff
@@ -11,6 +11,12 @@
...
 //= require rails-ujs
+//= require materialize-sprockets
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+
+document.addEventListener('DOMContentLoaded', function() {
+  var elems = document.querySelectorAll('.sidenav');
+  var instances = M.Sidenav.init(elems, options);
+});
```
Comment (diegosaul402) [inline app/assets/javascripts/application.js]: The `application.js` is only for importing other JS files, can't have any logic here, please move this to a new file and require it here.

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -22,6 +26,10 @@
         <%= link_to 'Login', new_user_session_path, class: 'navbar-link'  %>
       <% end %>
     </p>
+    
     <%= yield %>
+    <%= render(partial: 'footer') %>
```
Comment (diegosaul402) [inline app/views/layouts/application.html.erb]: These parentheses are not required, for better readability is better to remove them.

File: app/views/welcome/_footer.html.erb [hunk 6]
```diff
@@ -0,0 +1,26 @@
+<footer class="page-footer grey darken-4">
+  <div class="container">
+    <div class="row">
+      <div class="col l6 s12">
+        <h5 class="white-text">Footer Content</h5>
+        <p class="grey-text text-lighten-4">This is where the motto of finden will be</p>
+        <%= link_to image_tag('mh_logo.png', width: 100, height: 100), class: 'brand-logo'%>
```
Comment (diegosaul402) [inline app/views/welcome/_footer.html.erb]: Please add a space after `'brand-logo'`. Remember that every ERB tag has to have space at the beginning and the end. There is several code with this issue, please check all your changes before uploading them.

File: app/views/welcome/_footer.html.erb [hunk 7]
```diff
@@ -0,0 +1,26 @@
+<footer class="page-footer grey darken-4">
+  <div class="container">
+    <div class="row">
+      <div class="col l6 s12">
+        <h5 class="white-text">Footer Content</h5>
+        <p class="grey-text text-lighten-4">This is where the motto of finden will be</p>
```
Comment (diegosaul402) [inline app/views/welcome/_footer.html.erb]: We're using rails internationalization now, please move this text to a locale file.

Comment (archisss) [reply app/views/welcome/_footer.html.erb]: first i want to finish this commit and then i can add the i18n library

**Pull request merged**